src/Suggestor/Suggestor.ts

```diff
@@ -497,13 +498,28 @@ export function onlySuggestIfBracketOpen(fn: SuggestionBuilder, brackets: [strin
  *
  * This checks for simple pre-conditions:
  *  - Is the global filter (if set) in the line?
- *  - Is the line a task line (with a checkbox)
+ *  - Is the line a task line (with a checkbox)?
+ *  - OR is the line the first line of a Kanban card?
  * @param line
- * @param cursorPosition - the cursor position, when 0 is presumed to mean 'at the start of the line'.
- *                          See 'ch' in https://docs.obsidian.md/Reference/TypeScript+API/EditorPosition
+ * @param cursor - the cursor position, when ch is 0 it is presumed to mean 'at the start of the line'.
+ *                          See https://docs.obsidian.md/Reference/TypeScript+API/EditorPosition
+ * @param editor - the editor instance to which the suggest belongs
  */
-export function canSuggestForLine(line: string, cursorPosition: number) {
-    return GlobalFilter.getInstance().includedIn(line) && cursorIsInTaskLineDescription(line, cursorPosition);
+export function canSuggestForLine(line: string, cursor: EditorPosition, editor: Editor) {
+    return (
+        GlobalFilter.getInstance().includedIn(line) &&
+        (cursorIsInTaskLineDescription(line, cursor.ch) || cursorIsInKanbanDescription(cursor, editor))
+    );
+}
+
+/**
+ * Return true if the cursor is on the first line of a Kanban card
+ *
+ * @param cursor - the cursor position
+ * @param editor - the editor instance to which the suggest belongs
+ */
+function cursorIsInKanbanDescription(cursor: EditorPosition, editor: Editor) {
```

Comment:
I am rather concerned that this PR does not scale well for enabling other plugins to use the same functionality, and it requires logic in Tasks that is specific to the Kanban plugin.

I would like to explore ways of implementing this that are not tied to Kanban.

For example, how about something like `editorComponent?.showTasksPluginAutoSuggest(cursor, editor)`...

Then the logic goes in that method in Kanban - and can be also used later in any other plugin that wishes to expose this capability, rather than accumulating logic for other plugins inside Tasks.